content/overview/pricing/pricing-terminology.md

```diff
@@ -23,7 +23,7 @@ A system maps to an [“org” in cloud.gov and Cloud Foundry](http://docs.cloud
 
 ### Support {#support}
 
-“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement. We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
+“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement (SLA). We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
```

Comment:
Agreed on the addition of the abbreviation. However, now that I'm looking at this - shouldn't this be phrased differently? 'We have no existing SLA' doesn't seem quite right. Shouldn't it be something more along the lines of: We currently can not offer Service Level Agreements? @brittag